relations/services/model.py

```diff
@@ -20,4 +20,12 @@ class RelationDB(db.Model):
     description = Column(Text(1024))
     added_at = Column(DateTime)
     creator = Column(String(255), nullable=True)
-    supercedes_or_suppresses = Column(Integer, nullable=True)
+    supercedes_or_suppresses = Column(String(255), nullable=True)
```

Comment:
Why make this a ``String`` column? Seems like we could go further by adding a ``ForeignKey`` constraint, so that it refers to a ``RelationDB.id``.

Reply:
Because I'm a bit hesitant to use the integer index of as the `RelationID` directly.  An ID of e-print is like `1234.56789`, which must be handled as a string and makes other types of ID format available. I think it's better to use a string format to preserve extensibility.

Reply:
I'll fix it to `Integer` and `ForeignKey`.